crates/hc/README.md

```diff
@@ -4,12 +4,15 @@ Provides the `hc` binary, a helpful CLI tool for working with Holochain.
 
 ```shell
 $ hc -h
-
-holochain_cli 0.1.0
+holochain_cli 0.1.3
 Holochain CLI
 
-Work with DNA and hApp bundle files, set up sandbox environments for testing and development purposes, make direct admin
-calls to running conductors, and more.
+Work with DNA, hApp and web-hApp bundle files, set up sandbox environments for testing and development purposes, make
+direct admin calls to running conductors, and more.
+
+EXTENSIONS:
```

Comment:
This will only be there if the hc-launch and hc-scaffold binaries are also installed in the system. So we could drop the EXTENSIONS part if we want to.